packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import cx from 'classnames'
+import { TeamsProcessedSvgIconSpec } from '../types'
+import { teamsIconClassNames } from 'src/themes/teams/components/Icon/svg'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <g className={cx(teamsIconClassNames.filled, classes.outlinePart)}>
```

Comment:
this should be `teamsIconClassNames.outline`

Reply:
sorry but why?

Reply:
this is needed to add the correct static class for outline parts of the svg